Modules/_hashopenssl.c

```diff
@@ -1133,9 +1234,9 @@ pbkdf2_hmac_impl(PyObject *module, const char *hash_name,
     char *key;
     long dklen;
     int retval;
-    const EVP_MD *digest;
+    PY_EVP_MD *digest = NULL;
```

Comment:
This assignement looks useless, just declare the type on the same line which sets it?

```
PY_EVP_MD *digest = py_digest_by_name(module, hash_name, Py_ht_pbkdf2);
```

Reply:
SGTM